content/en/docs/concepts/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Concepts
-description: The collection of requirements and scenarios that define v2 of the Notary project
+description: The collection of requirements and scenarios that define the Notary Project
```

Comment:
```suggestion
description: The collection of requirements and scenarios that define The Notary Project
```
Wondering if we want to use "The Notary Project" or "the Notary Project"

Reply:
fixed